fem/fespace.hpp

```diff
@@ -144,8 +144,9 @@ class FiniteElementSpace
    static inline int DecodeDof(int dof, double& sign)
    { return (dof >= 0) ? (sign = 1, dof) : (sign = -1, (-1 - dof)); }
 
-   /// Helper to get vertex, edge or face DOFs (entity=0,1,2 resp.).
+   /// Helpers to get vertex, edge or face DOFs (entity=0,1,2 resp.).
    void GetEntityDofs(int entity, int index, Array<int> &dofs) const;
+   void GetDegenerateFaceDofs(int index, Array<int> &dofs) const;
```

Comment:
I don't know what is meant by "degenerate face dofs".  A bit of explanation would be useful here or a reference to documentation elsewhere if that makes more sense.

Reply:
Yes, this one is tricky, added an explanation inside the method.